js/min/editor-min.js

```diff
@@ -437,10 +438,49 @@ editor.on('init', function(e) {
 		if ( wrapper === null ) {
 			return;
 		}
+
+		if ( wrapper.find('.pbs-align-left').length === 0 ) {
+			$('<div class="dashicons dashicons-align-left pbs-align-left" data-mce-bogus="1" title="Align left"></div>').insertBefore( wrapper.find('.toolbar > .dashicons:eq(-1)') );
+		}
+
+		if ( wrapper.find('.pbs-align-center').length === 0 ) {
+			$('<div class="dashicons dashicons-align-center pbs-align-center" data-mce-bogus="1" title="Align center"></div>').insertBefore( wrapper.find('.toolbar > .dashicons:eq(-1)') );
+		}
+
+		if ( wrapper.find('.pbs-align-right').length === 0 ) {
+			$('<div class="dashicons dashicons-align-right pbs-align-right" data-mce-bogus="1" title="Align right"></div>').insertBefore( wrapper.find('.toolbar > .dashicons:eq(-1)') );
+		}
 		
 		if ( wrapper.find('.clone').length === 0 ) {
 			$('<div class="dashicons dashicons-images-alt clone" title="Clone"></div>').insertBefore( wrapper.find('.toolbar > .dashicons:eq(-1)') );
 		}
+
+		if ( $(e.target).is('.dashicons.dashicons-align-left') ) {
+			wrapper.addClass('pbs-alignleft');
```

Comment:
`addClass` and `removeClass` can take multiple class names. The code here can be shortened.